lib/reducers/create-otp-reducer.js

```diff
@@ -49,6 +51,38 @@ const MAX_RECENT_STORAGE = 5
 
 // TODO: fire planTrip action if default query is complete/error-free
 
+const requiredConfigKeys = [
+  'geocoder.type'
+]
+
+function validateInitalState (initialState) {
+  const {config} = initialState
+
+  const errors = []
+
+  // iterate through required keys to ensure integrity of config
+  // TODO: replace with JSON Schema validation, probably in a script that lives outside of the app
```

Comment:
Maybe this TODO should be elevated to the function def? Also, please give this function js doc.